packages/autocomplete-core/propGetters.ts

```diff
@@ -33,6 +33,44 @@ export function getPropGetters({
     };
   };
 
+  const getFormProps: GetFormProps = rest => {
+    return {
+      onSubmit: event => {
+        event.preventDefault();
+
+        // @TODO: call the `onInputChange` or `onSubmit` user prop?
```

Comment:
I'm thinking about the InstantSearch use case here.

Imagine we replace the default `searchBox` widget with this autocomplete library search box. We can add a top-level option `onSelect` or `onSubmit` that would allow to trigger a search (by calling `refine` of `connectSearchBox`).

We can revisit this idea later.

Reply:
Yeah, that seems to be a good way to connect with InstantSearch. Let's revisit this later when we actually try to implement the integration.